### PR TITLE
Add Cycle7 

### PR DESCRIPTION
**Pull request by Vertex-kb**
## Description
Add Cycle7 

## QMK Pull Request
https://github.com/qmk/qmk_firmware/pull/23290

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have formatted the JSON file to have consistent formatting with the rest of the repository.
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Please remove V2 version (src/ folder one) as this is legacy and we do not accept anymore pull requests for them.

**Comment by Vertex-kb** [comment]
Hello, the firmware at that time was version V2, and these PCBs have already been in the hands of users. Can you be accommodating?

**Comment by Xelus22** [comment]
> Hello, the firmware at that time was version V2, and these PCBs have already been in the hands of users. Can you be accommodating?

Your QMK PR was only 2 months ago, V2 has been deprecated for at least 1 year already.

File: src/vertex/cycle7/cycle7.json
Comment (dunk2k) [inline src/vertex/cycle7/cycle7.json:1]: as per comments, please delete this file

File: v3/vertex/cycle7/cycle7.json [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  "menus": [
+    "qmk_rgblight"
+  ],
+"layouts": {
+"labels":[
+      "Split Backspace",
+      "ISO Enter",
+      "Split Left Shift",
+      "Split Right Shift",
+["Bottom Row", "Ansi", "Tsangan","Wkl"]
```
Comment (dunk2k) [inline v3/vertex/cycle7/cycle7.json:18]: please format this section of code

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Comment by Cipulot** [comment]
VIA Keymap moved after last QMK Breaking Changes: https://github.com/the-via/qmk_userspace_via/tree/main/keyboards/vertex/cycle7/keymaps/via

The PR opener should delete the V2 defs since they are not needed nor used and apply the formatting of the json in V3.

**Pull request closed without merging**